### ISSUE TITLE
Consider allowing to configure two thresholds instead of one

### ISSUE THREAD

**Issue by rimas-kudelis** [alaneuler/batteryKid#20]
It would be nice if I could configure two thresholds, for example:
- stop charging when reached 80% or above
- start charging when reached 50% or below

AFAIK, it's quite unhealthy for the battery to jump between charging and discharging modes, which is why a scenario like above might be desired.

From the current UI it looks like the app will just attempt to maintain my battery at exactly 70%.

**Comment by alaneuler** [comment]
The limit deviation is what you want. It defaults to 2, which means (with threshold set to 70):
1. start discharging when above 72%
2. start charging when below 68%
3. onhold between 68% and 72%

In your case, you can set threshold to 65% and limit deviation to 15.

**Comment by rimas-kudelis** [comment]
I thought so! Somehow I missed that field on the first day I started using the app. However, having to choose the middle of the interval and then the deviation seems more complex than just being able to choose the beginning and end of the interval. It's two fields either way, but the latter method doesn't involve calculation. :)

**Comment by alaneuler** [comment]
> I thought so! Somehow I missed that field on the first day I started using the app. However, having to choose the middle of the interval and then the deviation seems more complex than just being able to choose the beginning and end of the interval. It's two fields either way, but the latter method doesn't involve calculation. :)

Yeah, you're right to this point but this may involve some work to change the UI.
